analyzers/tests/SonarAnalyzer.Test/TestCases/ConditionalStructureSameImplementation_If.cs

```diff
@@ -127,6 +127,45 @@ public void Exception(int a)
                 DoSomething1();
             }
         }
+
+        public bool FalseNegative(string s, out int kind)
```

Comment:
I would give this method a better name than `FalseNegative`, explaining what the test scenario is about.
For example something like `ElseIfChain`.